content/en/blog/2020-10-30-charts-repo-deprecation.md

```diff
@@ -0,0 +1,38 @@
+---
+title: "Helm Chart Repository Deprecation Update"
+slug: "charts-repo-deprecation"
+authorname: "Vic Iglesias"
+author: "@vicnastea"
+authorlink: "https://twitter.com/vicnastea"
+date: "2020-10-30"
+---
+
+Back in 2019, when the Helm v2 support timeline and end of life plan was announced, [the deprecation](https://github.com/helm/charts#deprecation-timeline) of the [helm/charts GitHub repository](https://github.com/helm/charts) was announced, as well. The primary reason for the deprecation is the significant increase in upkeep for the [repo maintainers](https://github.com/helm/charts/blob/master/OWNERS). Over the last couple of years the number of charts under maintainance increased from ~100 to 300+ causing a commesurate increase in pull requests and updates to the repo, unfortunately despite many efforts to automate review and maintainance tasks the amount of time available from maintainers has not increased in unison.
+
+When we announced the deprecation we also began to share the tools and guidance that we had used to maintain the helm/charts repo. For folks that want to host and maintain their own repositories you now have these tools available to streamline the process:
+
+- [Chart Testing](https://github.com/helm/chart-testing) provides linting and testing for PRs against your charts
+- [Chart Releaser](https://github.com/helm/chart-releaser) provides tooling to help you host your own chart repo with GitHub Relases and Pages used to host your artifacts
+- [Testing and Releasing Github Actions](https://github.com/helm?q=chart+action) to automate the tooling described above using GitHub Actions
+
+With these tools available we've enabled [many charts](https://github.com/helm/charts/issues/21103) to migrate to their own repositories for active maintainence.
+
+## Key Dates and Recommended Actions
+
+There has been refinement to the plans and confusion/questions about what happens next, so we wanted to provide a timeline of key events and **recommended actions** moving forward:
+
+* Nov 2, 2020 - READMEs for all non-deprecated charts will get a note added, stating that they will no longer be updated
+    * **RECOMMENDED ACTION** - If you depend on a chart in the Charts repository look for the new maintainance location. If one does not exist consider adopting the chart.
```

Comment:
> maintainance

This is a typo, but also I think the sentence may make more sense if you say "new official location"?

Reply:
Also, I suggest a comma between `exist` and `consider`.